elasticsearch.go

```diff
@@ -4,7 +4,7 @@ import (
 	"encoding/json"
 	"github.com/buger/elastigo/api"
 	"github.com/buger/elastigo/core"
-	"github.com/buger/gor/proto"
```

Comment:
Unless we also vendor gor, that would then make us vulnerable to upstream changes?

I also talked to the gor author and he thought that firehose output was too specialized to be merged back so I wasn't prioritizing that. Maybe I should try harder to keep that option open? There are also other changes here that make it more valuable to us, but a bit more specialized so less useful upstream (e.g. kayvee logging). Unsure where to balance on this